smithy-aws-protocol-tests/model/shared-types.smithy

```diff
@@ -120,3 +120,23 @@ structure GreetingStruct {
 list GreetingList {
     member: GreetingStruct
 }
+
+intEnum IntegerEnum {
+    A = 1
+    B = 2
+    C = 3
+}
+
+list IntegerEnumList {
+    member: IntegerEnum
+}
+
+@uniqueItems
+list IntegerEnumSet {
+     member: IntegerEnum
+}
+
+map IntegerEnumMap {
+     key: String,
+     value: IntegerEnum
+}
```

Comment:
Nit: you can and should configure your tools to automatically ensure the file ends with a newline. iirc vim and vscode do this by default, but you have to change this yourself in jetbrains